src/lib/packet-print.c

```diff
@@ -951,6 +955,9 @@ pgp_sprint_pubkey(const pgp_key_t *key, char *out, size_t outsize)
                        BN_bn2hex(key->key.pubkey.key.rsa.e));
         break;
     case PGP_PKA_EDDSA:
+        cc += snprintf(
```

Comment:
it was removed by later commit. you can remove it (case for PGP_PKA_EDDSA and ...ECDSA is the same)